### ISSUE TITLE
Matching data together with line continuations

### ISSUE THREAD

**Issue by elfring** [comby-tools/comby#310]
The tool “Comby” manages [a bit of background information for selected data structures](https://github.com/comby-tools/comby.dev/blob/461f661eb1a4f06dc80d4c6ae446d404ae1dcd9c/website/docs/overview.md#does-it-work-on-my-language "Does it work on my language?").
I find [the available software documentation](https://github.com/comby-tools/comby.dev/blob/461f661eb1a4f06dc80d4c6ae446d404ae1dcd9c/website/docs/syntax-reference.md#match-and-rewrite-syntax "Match syntax") improveable for the detail if (and how) [line continuations](https://en.wikipedia.org/wiki/Comparison_of_programming_languages_(syntax)#Line_continuation "Description for the usage of line continuations") can be supported for desirable data processing.
:crystal_ball: Would you like to extend the possibilities accordingly?

**Comment by rvantonder** [comment]
Yeah good point, matching line continuations would be nice to have :-) Let's keep this issue open to track it. Unfortunately I don't have a ton of time on my hands so this probably won't happen soon though.
